Scripts/loginSignup.js

```diff
@@ -0,0 +1,12 @@
+const container = document.querySelector(".formContainer");
+const switchButtons = document.querySelectorAll(".pointer");
+
+switchButtons.forEach(element => {
+    element.addEventListener("click",() => {
+        container.classList.toggle("sign-in");
+        container.classList.toggle("sign-up");
+    });
+    console.log("da")
```

Comment:
is there any reason for console.log here ?

Reply:
oh, Thanks for pointing out 
i missed it